gallery/gallery/lib/l10n/intl_en_US.arb

```diff
@@ -665,6 +665,124 @@
+  },
+  "demoSectionedMenuTitle": "Sectioned menu",
+  "@demoSectionedMenuTitle": {
+    "description": "Title for the sectioned menu component demo."
+  },
+  "demoSimpleMenuTitle": "Simple menu",
+  "@demoSimpleMenuTitle": {
+    "description": "Title for the simple menu component demo."
+  },
+  "demoChecklistMenuTitle": "Checklist menu",
+  "@demoChecklistMenuTitle": {
+    "description": "Title for the checklist menu component demo."
+  },
+  "demoMenuSubtitle": "Menu buttons and simple menus",
+  "@demoMenuSubtitle": {
+    "description": "Short description for the sliders component demo."
+  },
+  "demoMenuDescription": "A menu displays a list of choices on a temporary surface. They appear when users interact with a button, action, or other control.",
+  "@demoMenuDescription": {
+    "description": "Description for the menu demo."
+  },
+  "demoMenuItemValueOne": "Menu item value one",
+  "@demoMenuItemValueOne": {
+    "description": "The menu number one."
+  },
+  "demoMenuItemValueTwo": "Menu item value two",
+  "@demoMenuItemValueTwo": {
+    "description": "The menu item number two"
+  },
+  "demoMenuItemValueThree": "Menu item value three",
+  "@demoMenuItemValueThree": {
+    "description": "The menu item number three."
+  },
+  "demoMenuOne": "One",
+  "@demoMenuOne": {
+    "description": "The number one."
+  },
+  "demoMenuTwo": "Two",
+  "@demoMenuTwo": {
+    "description": "The number two."
+  },
+  "demoMenuThree": "Three",
+  "@demoMenuThree": {
+    "description": "The number three."
+  },
+  "demoMenuFour": "Four",
+  "@demoMenuFour": {
+    "description": "The number four."
+  },
+  "demoMenuAnItemWithAContextMenuButton": "An item with a context menu button",
+  "@demoMenuAnItemWithAContextMenuButton": {
+    "description": "Label next to a button that opens a menu. A menu displays a list of choices on a temporary surface. Used in as an example in a demo."
+  },
+  "demoMenuContextMenuItemOne": "Context menu item one",
+  "@demoMenuContextMenuItemOne": {
+    "description": "Text label for a context menu item. A menu displays a list of choices on a temporary surface. Used in as an example in a demo."
+  },
+  "demoMenuADisabledMenuItem": "A disabled menu item",
+  "@demoMenuADisabledMenuItem": {
+    "description": "Text label for a disabled menu item. A menu displays a list of choices on a temporary surface. Used in as an example in a demo."
+  },
+  "demoMenuContextMenuItemThree": "Context menu item three",
+  "@demoMenuContextMenuItemThree": {
+    "description": "Text label for a context menu item three. A menu displays a list of choices on a temporary surface. Used in as an example in a demo."
+  },
+  "demoMenuAnItemWithASectionedMenu": "An item with a sectioned menu",
+  "@demoMenuAnItemWithASectionedMenu": {
+    "description": "Label next to a button that opens a sectioned menu . A menu displays a list of choices on a temporary surface. Used in as an example in a demo."
+  },
+  "demoMenuPreview": "Preview",
```

Comment:
Give a better description to these 4 so that each word is properly described as if it was a real button